src/main/groovy/life/qbic/portal/sampletracking/web/views/ControlElements.groovy

```diff
@@ -52,12 +48,9 @@ class ControlElements extends VerticalLayout {
         // Add menu allowing status selection for new sample
         statusSelectMenu = new NativeSelect<Status>("New Sample Status")
         statusSelectMenu.setEmptySelectionAllowed(false)
-        statusSelectMenu.setItems(Status.values())
-
-        /* For Test purposes
-        //TODO: remove in release
-        sampleSelectMenu.setItems("Waiting", "Processing", "Processed")
-        */
+        ArrayList<Status> filteredStatusList = new ArrayList<Status>([Status.DATA_AT_QBIC, Status.METADATA_REGISTERED])
+        ArrayList<Status> selectableStatusList = getSelectableStatusList(filteredStatusList)
+        statusSelectMenu.setItems(selectableStatusList)
```

Comment:
Seems very complicated. Maybe set a static variable 
`final static List<Status> FORBIDDEN_STATUS_OPTIONS = new ArrayList(Status.DATA_AT_QBIC, Status.METADATA_REGISTERED)`
Then you can simply do
```suggestion
List<Status> selectableStatusOptions = Status.values().findall { status ->
  !(status in FORBIDDEN_STATUS_OPTIONS)
}
statusSelectMenu.setItems(selectableStatusOptions)
```